### DISCUSSION TITLE
I am Bad

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#614] [(A) Anything]
Okay so i tryed to do discord but there is a hack i know on how to bypass schools dumb filter but its stuck on this screen
![1F719197-70B1-4799-9333-172A03B175D2](https://user-images.githubusercontent.com/62729648/110048072-f09b1000-7d03-11eb-813b-a1f3ee25d746.png)

**Comment by ghost** [comment]
waht is this "hack"?

**Reply by ghost** [reply]
yeah

**Comment by ghost** [comment]
just use a VPN when my school blocks a site I use earth VPN and bypass it

**Comment by ghost** [comment]
or just go to canary.discord.com

**Reply by ArcerionDev** [reply]
canary.discord.com is a subdomain of discord.com so it'll still be blocked
